kedro/framework/cli/starters.py

```diff
@@ -170,13 +170,13 @@ def starter():
 @click.option("--starter", "-s", "starter_alias", help=STARTER_ARG_HELP)
 @click.option("--checkout", help=CHECKOUT_ARG_HELP)
 @click.option("--directory", help=DIRECTORY_ARG_HELP)
-@click.option("--addons", "-a", "selected_add_ons_flag", help=ADDON_ARG_HELP)
+@click.option("--tools", "-a", "selected_tools", help=TOOLS_ARG_HELP)
```

Comment:
```suggestion
@click.option("--tools", "-t", "selected_tools", help=TOOLS_ARG_HELP)
```

Reply:
Good catch! I'll be going through those next